Remove ldconfig(1) execution to prevent the installation script from accessing
outside of a sandbox.

diff --git a/install-template.sh b/install-template.sh
index 11bf392..bc579f5 100644
--- a/src/rust-installer/install-template.sh
+++ b/src/rust-installer/install-template.sh
@@ -697,17 +697,6 @@ done
 # Drop the version number into the manifest dir
 echo "$TEMPLATE_RUST_INSTALLER_VERSION" > "${ABS_LIBDIR}/${TEMPLATE_REL_MANIFEST_DIR}/rust-installer-version"
 
-# Run ldconfig to make dynamic libraries available to the linker
-if [ "$CFG_OSTYPE" = "unknown-linux-gnu" -a ! -n "$CFG_DISABLE_LDCONFIG" ]; then
-    msg "running ldconfig"
-    ldconfig
-    if [ $? -ne 0 ]
-    then
-        warn "failed to run ldconfig."
-        warn "this may happen when not installing as root and may be fine"
-    fi
-fi
-
 # Sanity check: can we run the installed binaries?
 #
 # As with the verification above, make sure the right LD_LIBRARY_PATH-equivalent
